src/style.css

```diff
@@ -0,0 +1,126 @@
+@font-face {
+  font-family: Avenir;
+  src: url(./fonts/Avenir-Heavy.ttf);
+}
+
+@font-face {
+  font-family: Avenir;
+  src: url(./fonts/Avenir-Book.ttf);
+}
+
+@font-face {
+  font-family: Avenir;
+  src: url(./fonts/Avenir.ttc);
+}
+
+html {
+  font-family: Avenir, sans-serif;
+  font-weight: 600;
+  box-sizing: border-box;
+}
+
+body {
+  margin: 0;
+  padding: 20px 7.6px 7.5px ;
+}
+
+ul {
+  margin: 0;
+}
+
+form {
+  position: relative;
+}
+
+.bar1 {
+  width: 100%;
+  font-family: "Avenir", sans-serif;
+  font-size: 16px;
+  line-height: 22px;
+  outline: none;
+  padding-left: 62px;
+  height: 70px;
+  margin-bottom: 20px;
+  background: #fff;
+  border: 1px solid #e1e7ed;
+  box-shadow: 0 1px 8px rgba(61, 78, 97, 0.1);
+  border-radius: 4px;
+}
+
+.bar1:focus {
+  background: linear-gradient(180deg, #fff 0%, #f6f6f7 100%);
+  border: 1px solid #e1e7ed;
+  border-radius: 3px;
+  /* filter: drop-shadow(0 4px 4px rgba(0, 0, 0, 0.25)); */
+  font-weight: 300;
+}
+
+.bar1:hover {
+  background: #fff;
+  border: 1px solid #e1e7ed;
+  box-shadow: 0 3px 8px rgba(61, 78, 97, 0.2);
+  border-radius: 4px;
+}
+
+.bar1::placeholder {
+  font-family: "Avenir", sans-serif;
+  font-size: 16px;
+  line-height: 22px;
+  color: #3d4e61;
```

Comment:
the placeholder should inherit styles from the main selector. try to remove these styles, result should be the same